modules/peers.js

```diff
@@ -236,6 +238,17 @@ __private.getMatched = function(test, peers) {
 	return peers.filter(peer => peer[key] === value);
 };
 
+/**
+ * Check if the ip exists in the peer blacklist coming from config file.
+ *
+ * @param ip
+ * @returns {boolean}
+ * @todo Add description for the params and the return value
+ */
+__private.isBlacklisted = function(ip) {
+	return self.blackListPeers.indexOf(ip) > -1;
```

Comment:
since we are using latest javascript we can start using self.blackListPeers.includes(ip);

Reply:
We should do this in a wider way. I see many other cases in the code.